### ISSUE TITLE
bug(mat-autocomplete): Programmatically focused input doesn't respond to input when using NVDA

### ISSUE THREAD

**Issue by dwaynelavon** [angular/components#21652]
#### Reproduction

Stackblitz: https://stackblitz.com/edit/angular-scnfuz-tonvzm?file=src%2Fapp%2Fautocomplete-simple-example.ts

Steps to reproduce:
1. Enable NVDA
2. Open Stackblitz link in Firefox
3. Focus autocomplete input and select an option
4. Tab to the delete button on that option's newly created card
5. Press enter to delete and observe focus on autocomplete input
6. Try to enter text
 
 
#### Expected Behavior

What behavior were you expecting to see?
After card is removed, input should be focused, its label should be announced, and it should accept input.

#### Actual Behavior

What behavior did you actually see?
Autocomplete input is focused but labels are not announced and it does not accept input.

#### Environment

- Angular: 11.0.3
- CDK/Material: 11.0.3
- Browser(s): Firefox
- Operating System (e.g. Windows, macOS, Ubuntu): Windows

**Comment by crisbeto** [comment]
I believe that this is the same root cause as https://github.com/angular/components/issues/20761. It seems like moving focus programmatically ends up putting the screen reader in navigation mode which prevents you from typing. I'm not sure that we can do anything about this, but let's leave it open in case somebody has more information.

**Comment by zarend** [comment]
Hello @dwaynelavon ,

Thank you for reporting. I believe we may have fixed this issue in v15. I was wondering if we could check with you if this issue requires further action. Does upgrading to v15 resolve the issue for you?

Thanks,
Zach

**Comment by zarend** [comment]
Hello, 

Thank you for reporting this issue. I cannot reproduce on Firefox with the provided link (https://stackblitz.com/edit/angular-scnfuz-tonvzm?file=src%2Fapp%2Fautocomplete-simple-example.ts). Unfortunately, we are not able to take action if we cannot reproduce it. This may be a quirk with how NVDA works and not an issue with the component.  If there are still issues with Select component, please reopen this issue or file a new issue report.

Environment
 - NVDA Version 2022.3
 - Firefox Version 109.0 (64-bit)
 - Windows 10 Enterprise version 21H2
 
 -Zach
